packages/authgear-react-native/index.d.ts

```diff
@@ -954,12 +1041,15 @@ export declare interface ReauthenticateOptions {
  */
 export declare interface ReauthenticateResult {
   /**
-   * UserInfo.
+   * The user info after reauthenticated.

```

Comment:
does it mean "The updated user info after reauthentication"?

Reply:
Oh yes yes